### ISSUE TITLE
select list

### ISSUE THREAD

**Issue by StewBash** [twbs/bootstrap#24935]
Before opening an issue:

- [Search for duplicate or closed issues](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [Validate](https://html5.validator.nu/) and [lint](https://github.com/twbs/bootlint#in-the-browser) any HTML to avoid common problems
- Prepare a [reduced test case](https://css-tricks.com/reduced-test-cases/) for any bugs
- Read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md)

When asking general "how to" questions:

- Please do not open an issue here
- Instead, ask for help on [StackOverflow, IRC, or Slack](https://github.com/twbs/bootstrap/blob/master/README.md#community)

When reporting a bug, include:

- Operating system and version (Windows, Mac OS X, Android, iOS, Win10 Mobile)
- Browser and version (Chrome, Firefox, Safari, IE, MS Edge, Opera 15+, Android Browser)
- Reduced test cases and potential fixes using [CodePen](https://codepen.io/) or [JS Bin](https://jsbin.com/)

When suggesting a feature, include:

- As much detail as possible for what we should add and why it's important to Bootstrap
- Relevant links to prior art, screenshots, or live demos whenever possible

**Comment by StewBash** [comment]
Simple Select is broken... Dropdown list no longer works properly.

From Bootstrap 4 documentation: https://getbootstrap.com/docs/4.0/components/forms/

Checkout the select list boxes live... they do not work either!!!

**Comment by mdo** [comment]
They work for me. Closing due to lack of information.

**Comment by StewBash** [comment]
Sorry for not leaving more information, however a simple select list does not work though Microsoft Edge... It is working with Chriome however. 

I have been integrating Bootstrap 4 into Visual Studio 2017 .NET Core projects... I finally had a chance to evaluate the customization mechanisms of Bootstrap using NPM compiler locally... So far it looks good however I do not like the shadow-box on my forms controls. I was looking to remove the shadow-box all together or, make it smaller... 

In converting a Bootstrap 3.3.7 project, I started getting this behavior whereby a select list with options does not act like it did in 3.3.7... I get a solid 2px black box around the dropdown list... I started to evaluate from scratch and noticed that I get this issue simply going to your web site in Microsoft Edge browser... In Chrome everything is working fine...

Sorry for the lack of information... My apologies!
